Update filterbar.sass: use the whole screen for .fulltab

jbsilva opened the pull request:
This will make the gallery fill the whole screen

mfg92 commented:
On my side, the gallery was already filling up the whole screen. Can you explain what problem you were trying to solve?

With your changes, I am unable to scroll through the gallery.

jbsilva commented:
`position: fixed` is indeed wrong. I updated my PR.

But without `min-width`, the gallery was taking only a small part of the screen:

<img src="https://github.com/user-attachments/assets/3c5ba35a-1540-4044-b1a8-0a5b4f868795" width="300" />

mfg92 commented:
Ok, I was able to reproduce the problem you describe. This bug was hidden on my site due to the CSS of my theme.
Thanks for your contribution!
